ehrapy/anndata/_feature_specifications.py

```diff
@@ -86,13 +174,55 @@ def feature_type_overview(adata: AnnData):
         branch.add(date)
 
     branch = tree.add("📐[b] Numerical features")
-    for numeric in sorted(adata.var_names[adata.var[FEATURE_TYPE_KEY] == CONTINUOUS_TAG]):
+    for numeric in sorted(adata.var_names[adata.var[FEATURE_TYPE_KEY] == NUMERIC_TAG]):
         branch.add(numeric)
 
     branch = tree.add("🗂️[b] Categorical features")
     cat_features = adata.var_names[adata.var[FEATURE_TYPE_KEY] == CATEGORICAL_TAG]
     df = anndata_to_df(adata[:, cat_features])
-    for categorical in sorted(cat_features):
-        branch.add(f"{categorical} ({df.loc[:, categorical].nunique()} categories)")
+
+    if "encoding_mode" in adata.var.keys():
+        unencoded_vars = adata.var.loc[cat_features, "unencoded_var_names"].unique().tolist()
+
+        for unencoded in sorted(unencoded_vars):
+            if unencoded in adata.var_names:
+                branch.add(f"{unencoded} ({df.loc[:, unencoded].nunique()} categories)")
+            else:
+                enc_mode = adata.var.loc[adata.var["unencoded_var_names"] == unencoded, "encoding_mode"].values[0]
+                branch.add(f"{unencoded} ({adata.obs[unencoded].nunique()} categories); {enc_mode} encoded")
+
+    else:
+        for categorical in sorted(cat_features):
+            branch.add(f"{categorical} ({df.loc[:, categorical].nunique()} categories)")
 
     print(tree)
+
+
+def correct_feature_types(adata, features: list[str] | str, corrected_type: str):
```

Comment:
Should this be called `correct` or `replace`?

Should `features` be typed with `Iterable[str]`?

Reply:
Hmm... I don't really have a preference for the naming. Do you have a preference, @eroell?

Regarding the typing: Absolutely true, I changed it.

Reply:
I now renamed `correct_feature_types` to `replace_feature_types`.